src/projects/pull-request/get-comments.html

```diff
@@ -68,7 +68,6 @@
           computed: '_computeCommentsGhPath(project, number)'
         },
         number: String,
-        fbComments: Object,
```

Comment:
Yes, it is. But because it's the return from the firebase-document the linter complained.